assignments/assignment-6/public/index.html

```diff
@@ -68,6 +68,32 @@ <h3>Enter up to 4 Teams:</h3>
       </div>
       <div class="game-container">
         <div class="game-board">
+          <div class="question-window">
+            <div class="question-close">x</div>
+            <div class="question-field"></div>
+            <div class="show-answer-container">
+              <button class="show-answer" onclick = "showAnswer()">Show Answer</button>
```

Comment:
I think we should extract this out of our HTML to it's own jQuery event listener in scripts.js

Reply:
Yeah, I didn't notice that one, I'll go and throw it in the JS file.

Reply:
Actually, that onClick is completely useless and something I forgot to remove, lemme get rid of that.